app/src/common/shared/com/igalia/wolvic/ui/widgets/Windows.java

```diff
@@ -1048,8 +1048,10 @@ public void onSharedPreferenceChanged(SharedPreferences sharedPreferences, Strin
                 return;
 
             WindowWidget frontWindow = getFrontWindow();
-            frontWindow.getPlacement().translationZ = getFrontWindowTranslationZ();
-            updateViews();
+            if (frontWindow != null) {
```

Comment:
How can this be ever possible? In which situation we don't have a front window?

Reply:
I added it because `getFrontWindow()` may return null.

I don't think that is likely to actually happen (maybe on startup or because of a bug?) but it seemed safer than the previous code. There are several other places in that file where we check that the front window is not null.